pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1465,6 +1465,81 @@ def test_set_value_resize(self, float_frame):
         with pytest.raises(ValueError, match=msg):
             res._set_value("foobar", "baz", "sam")
 
+    def test_reindex_with_multi_index(self):
```

Comment:
@jbrockmendel we should create a test_multi.py in this dir (@ChrisRobo you can do this if you want here)

Reply:
we're moving towards organizing by method instead of by index subclass, so this will eventually live in a test_reindex file (there are lots of reindex tests in this file, so this is fine for the time being.  before long ill batch-move them)